### ISSUE TITLE
rosdep libftdipp-dev not resolving properly for Ubuntu 16.04

### ISSUE THREAD

**Issue by koenlek** [ros/rosdistro#11626]
It tries to install "libftdipp-dev" but I think it should be "libftdipp1-dev" on xenial. Should the rosdep in https://github.com/ros/rosdistro/blob/master/rosdep/base.yaml
be updated from this:

```
libftdipp-dev:
  fedora: [libftdi-c++-devel]
  ubuntu: [libftdipp-dev]
```

To e.g. this:

```
libftdipp-dev:
  fedora: [libftdi-c++-devel]
  ubuntu: 
    precise: [libftdipp-dev]
    trusty: [libftdipp-dev]
    vivid: [libftdipp-dev]
    wily: [libftdipp-dev]
    xenial: [libftdipp1-dev]
```

**Comment by wjwwood** [comment]
Please open a pull request to address the problem. You might want to @ mention whoever added it originally to see what they think.

Thanks!

**Comment by koenlek** [comment]
Good one. Having some busy days, but I plan to make the pull request when I get the time.

**Comment by koenlek** [comment]
Finally found some time ;)

**Comment by tfoote** [comment]
Resolved in #12866
